docker/ckan/setup/prerun.py

```diff
@@ -162,7 +162,7 @@ def create_sysadmin():
         command = ["ckan", "-c", ckan_ini, "user", "show", name]
 
         out = subprocess.check_output(command)
-        if b"User:None" not in re.sub(b"\s", b"", out):
+        if b"User:None" not in re.sub(rb"\s", b"", out):
```

Comment:
```suggestion
        if b"User:None" not in re.sub(b"\s", b"", out):
```
Esto no deberia ser necesario

Reply:
cuando ejecuto flake localmente me lo marca:
![image](https://github.com/user-attachments/assets/194dc08d-32b2-4f46-9539-a780b60ccd71)

Reply:
Lo paso a \\s y vemos